param/parameterized.py

```diff
@@ -10,6 +10,7 @@
 import random
 import numbers
 import operator
+from .serializer import JSONSerialization
```

Comment:
This one import breaks parameterized.py's longstanding status as a file that can be used standalone, with no other files (not even `__init__.py`).  The original idea was that people might want to make their own set of specific Parameter classes to use with Parameterized, e.g. something much smaller than the full list that we provide, and we wanted people to be able to do that and to work with Parameterized in general without necessarily bringing in the specific Parameter classes beyond the one actually defined in Parameterized (param.String).  We aren't using that capability ourselves, so losing that won't hurt us, but it's been a feature of parameterized.py for so many years that I'm reluctant to lose it now.

```suggestion
# Allow this file to be used standalone if desired, albeit without JSON serialization
try: 
   from .serializer import JSONSerialization
except ImportError:
   JSONSerialization = None
```

Reply:
Sure, happy to make this serialization file optional.

Reply:
Implemented in c1c3978